### DISCUSSION TITLE
Is the Sysmon dashboard only used on windows 10 or from within SO

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10486] [Unsupported Versions] [answered]
Hi, i was wondering if the sysmon dashboard can be used as a stand alone on windows AND as another service on the SOC?

thx

**Comment by cm-ops** [comment]
Not sure what you are asking, but here is a link to the docs for visualizing Sysmon https://docs.securityonion.net/en/2.3/sysmon.html?#visualizations

**Reply by iqworks** [reply]
thanks. I was wondering if the sysmon kibana was used on windows only as a stand alone application? I will read that link, thanks again.

**Reply by dougburks** [reply, marked as answer]
In theory, it is possible to manually build the Elastic stack on Windows, but we don't recommend or support it.
